include/z64.h

```diff
@@ -931,7 +931,7 @@ typedef struct {
     /* 0x38 */ DmaRequest dmaRequest;
     /* 0x58 */ OSMesgQueue loadQueue;
     /* 0x70 */ OSMesg loadMsg;
-    /* 0x74 */ s16 unk_74[2];
+    /* 0x74 */ s16 unk_74[2]; // context-specific data used by the current scene draw config
```

Comment:
this is pretty interesting, had no idea this was a thing. would a general name like `data` `roomData` `params` or something work?
if you dont want to name this now thats fine

Reply:
The purpose is clear (imo) but the use cases not so much, so I think we should wait for more documentation on stuff using it (especially scene draw configs)